src/richie/apps/core/fields/multiselect.py

```diff
@@ -23,10 +23,12 @@ def to_sentence(elements):
 
     """
     if len(elements) > 2:
-        return _("{:s} and {:s}").format(
-            ", ".join(map(str.lower, elements[:-1])), str.lower(elements[-1])
-        )
-    return _(" and ").join(map(str.lower, elements))
+        return (
+            _("{:s} and {:s}").format(
+                ", ".join(map(str, elements[:-1])), str(elements[-1])
+            )
+        ).lower()
+    return (_(" and ").join(map(str, elements))).lower()
```

Comment:
can you add tests that validate the behavior you are targetting?

Reply:
This ensures that the following tests succeed in :

tests/apps/courses/test_models_course_run.py
- test_models_course_run_get_languages_display_one_language
- test_models_course_run_get_languages_display_request
- test_models_course_run_get_languages_display_three_languages                                                                                 
- test_models_course_run_get_languages_display_two_languages

in tests/apps/courses/test_templates_course_detail.py
- test_templates_course_detail_cms_draft_content
- test_templates_course_detail_cms_published_content
- test_templates_course_detail_state_with_cta
- test_templates_course_detail_state_without_cta

in tests/apps/courses/test_templates_course_run_detail.py
test_templates_course_run_detail_cms_draft_content
test_templates_course_run_detail_cms_published_content
test_templates_course_run_detail_no_index
test_templates_course_run_detail_state_with_cta
test_templates_course_run_detail_state_without_cta

Reply:
Here's the error log without moving the lower to later:
```
File "/app/src/richie/apps/core/fields/multiselect.py" in func
  259.                         [choicedict.get(value, value) for value in getattr(self, name)]

File "/app/src/richie/apps/core/fields/multiselect.py" in to_sentence
  29.     return _(" and ").join(map(str.lower, elements))

File "/usr/local/lib/python3.6/site-packages/django/utils/functional.py" in __wrapper__
  109.                 return getattr(res, method_name)(*args, **kw)

Exception Type: TypeError at /en/courses/new-course/
Exception Value: descriptor 'lower' requires a 'str' object but received a '__proxy__'
```

Reply:
Ok sorry. I thought you changed the behavior. This should be explained in your commit message maybe.